jackson/src/test/java/io/gravitee/definition/jackson/api/ApiDeserializerTest.java

```diff
@@ -87,6 +89,7 @@ public void definition_reformatContextPath() throws Exception {
         Assert.assertFalse(api.getProxy().getVirtualHosts().iterator().next().isOverrideEntrypoint());
     }
 
+    @Ignore
```

Comment:
Why ?

Reply:
There is no `JsonMappingException` thrown, this should be checked in the business logic, not in the deserializer.